data/preset/news.yaml

```diff
@@ -42,7 +42,7 @@ defines:
     contents:
     - type: ContentGroup.DynamicTitle
       properties:
-        format-string: !translate "see more from %s"
+        format-string: !translate "More news from %s"
```

Comment:
Drive-by comment: normally we should avoid changing translatable strings after the feature freeze, since it invalidates translations that we may have already paid to have done. I don't know if that's the case now, but make sure to check before merging this.